### ISSUE TITLE
[HybridApp][Fire Clean Up] fix pod and npm caches 

### ISSUE THREAD

**Issue by grgia** [Expensify/App#54318]
See https://expensify.enterprise.slack.com/canvas/C085GD1ET8A

# Context
 > Very few internal engineers had a working local HybridApp build due to multiple issues. We suggest resolving the issues and then sending an action required email to set up HybridApp on all internal engineer machines.

# Problem 
There are some pod and npm caches that I think are preventing people from building (myself included)

**Comment by staszekscp** [comment]
@AndrewGable - Yes, this is exactly the error we've encountered after merging the [NotificationService PR](https://github.com/Expensify/Mobile-Expensify/pull/13293). I managed to fix that by re-adding `Pods_NotificationServiceExtension.framework` like in the attached video. The changes generated by this action were included in [this PR](https://github.com/Expensify/Mobile-Expensify/pull/13317).

**Comment by AndrewGable** [comment]
Ok but I think we reverted that PR, right?

**Comment by puneetlath** [comment]
Adding the `Fire-Cleanup` label here. From #fireroom-2024-12-17-hybridapp-crash

**Comment by greg-schroeder** [comment]
Back from the holidays! Hmm. Not sure what's up here - can you clarify what needs to be done here @AndrewGable? This one is a little hard to follow 😅

**Comment by greg-schroeder** [comment]
Not sure which project this should live under. I asked

**Comment by AndrewGable** [comment]
Checking into this but I think this might be good now.

**Comment by greg-schroeder** [comment]
@AndrewGable do you mean this can be closed?

**Comment by AndrewGable** [comment]
Yes, all good here.
